stripe-connect/src/main/java/com/stripe/android/connectsdk/EmbeddedComponentManager.kt

```diff
@@ -1,7 +1,170 @@
 package com.stripe.android.connectsdk
 
+import android.os.Parcelable
+import androidx.activity.ComponentActivity
+import androidx.annotation.ColorInt
 import androidx.annotation.RestrictTo
+import kotlinx.parcelize.Parcelize
 
 @PrivateBetaConnectSDK
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-class EmbeddedComponentManager internal constructor()
+class EmbeddedComponentManager internal constructor() {
+
+    // TODO MXMOBILE-2760 - replace with actual implementation and remove the @Suppress annotations
+    constructor(
+        @Suppress("UNUSED_PARAMETER") activity: ComponentActivity,
+        @Suppress("UNUSED_PARAMETER") configuration: Configuration,
+        @Suppress("UNUSED_PARAMETER") fetchClientSecret: FetchClientSecretCallback,
+    ) : this() {
+        throw NotImplementedError("Not yet implemented")
+    }
+
+    @PrivateBetaConnectSDK
+    fun presentAccountOnboarding() {
+        throw NotImplementedError("Not yet implemented")
+    }
+
+    @PrivateBetaConnectSDK
+    fun presentPayouts() {
+        throw NotImplementedError("Not yet implemented")
+    }
+
+    @PrivateBetaConnectSDK
+    fun update(
+        // TODO MXMOBILE-2507 - replace with actual implementation and remove the @Suppress annotations
+        @Suppress("UNUSED_PARAMETER") appearance: Appearance,
+    ) {
+        throw NotImplementedError("Not yet implemented")
+    }
+
+    @PrivateBetaConnectSDK
+    fun logout() {
+        throw NotImplementedError("Not yet implemented")
+    }
+
+    // Configuration
+
+    @PrivateBetaConnectSDK
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    @Parcelize
+    data class Configuration(
+        val publishableKey: String,
+    ) : Parcelable
+
+    // Appearance classes
+
+    @PrivateBetaConnectSDK
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    @Parcelize
+    data class Appearance(
```

Comment:
You may want to split `Appearance` out into it's own file since it's going to have a lot of properties on it. It's also okay to omit it from this PR since we have a ticket scheduled in a later sprint to address it:
https://jira.corp.stripe.com/browse/MXMOBILE-2507

Reply:
Good point. I have a few PRs lined up after this - I'll leave this for now and address in further PRs